serverpod_auth_2/lib/additional_data.dart

```diff
@@ -0,0 +1 @@
+typedef AdditionalData = Map<String, dynamic>;
```

Comment:
Developers will need to be careful to basically only put ids in here, so that they can not be tempered with on the client (eg. use `{"invitation_code": "asdf"}` instead of `{"team": "7"}`).

Else maybe we should facilitate signing of the data (though lookups would also be nicer to smaller payload), such that developers get secure defaults.